app/src/main/java/io/multy/viewmodels/WalletViewModel.java

```diff
@@ -104,13 +104,14 @@ public WalletRealmObject createWallet(String walletName) {
             DataManager dataManager = DataManager.getInstance();
 
             final int topIndex = Prefs.getInt(Constants.PREF_WALLET_TOP_INDEX, 0);
-            final int currency = NativeDataHelper.Currency.BTC.getValue(); //TODO implement choosing crypto currency using enum NativeDataHelper.CURRENCY
 
             if (!Prefs.getBoolean(Constants.PREF_APP_INITIALIZED)) {
                 FirstLaunchHelper.setCredentials("");
             }
 
-            String creationAddress = NativeDataHelper.makeAccountAddress(dataManager.getSeed().getSeed(), topIndex, Constants.ZERO, currency);
+            String creationAddress = NativeDataHelper.makeAccountAddress(dataManager.getSeed().getSeed(),
+                    topIndex, Constants.ZERO, NativeDataHelper.Blockchain.BLOCKCHAIN_BITCOIN.getValue(),
```

Comment:
stop using Constants.ZERO like constants. It's just a zero so write 0. Let's get rid of Igor's practice wasting code.